docs/src/docs/core/Stepper.mdx

```diff
@@ -83,6 +83,12 @@ Loader will replace step icon. You can configure default loader in [MantineProvi
 
 <Demo data={StepperDemos.loading} />
 
+## Completed step
+
+You can use `Stepper.Completed` component to add a completed step, which content will be shown once all steps are completed.
+
+<Demo data={StepperDemos.completedStep} />
+
```

Comment:
I was not sure if we wanted to promote this functionality with its own section, or if the `Usage` section would be enough. In doubt, I added this. I can remove it if it's deemed unnecessary.

Reply:
I think we should add this to usage and remove this section

Reply:
Removed :) 

The usage example has been modified to use the new component already.